src/sbp_utils.c

```diff
@@ -294,7 +294,7 @@ void unpack_ephemeris(const msg_ephemeris_t *msg, ephemeris_t *e)
   e->sid              = sid_from_sbp(msg->sid);
   e->kepler.iode      = msg->iode;
   e->kepler.iodc      = msg->iodc;
-  e->fit_interval     = 4 * 60 * 60; /* TODO: this is a work around until SBP updated */
+  e->fit_interval     = (u8)(4 * 60 * 60); /* TODO: this is a work around until SBP updated */
```

Comment:
My bad. This was needed to pass compilation stage before I rebased to piksi_firmware/master. It is not needed anymore. I will remove this code change.